website/content/docs/networking/cni.mdx

```diff
@@ -1,93 +1,62 @@
+- You are familiar with [CNI plugins](https://www.cni.dev/plugins/current/).
+- You are familiar with [how Nomad supports Container Network Interface (CNI) plugins](/nomad/docs/networking/index.mdx#container-network-interface-plugins).
+- You are running Nomad on Linux.
 
-To use Nomad's [`transparent_proxy`][] feature, you will also need the
-[`consul-cni`][] plugin.
 
-## CNI plugins
+## CNI plugins and bridge networking workflow
 
-Spec-compliant plugins should work with Nomad, however, it's possible a plugin
-vendor has implemented their plugin to make non-standard API calls, or it is
-otherwise non-compliant with the CNI specification. In those situations the
-plugin may not function correctly in a Nomad environment. You should verify
-plugin compatibility with Nomad before deploying in production.
+Perform the following on each Nomad client:
 
-CNI plugins are installed and configured on a per-client basis. Nomad consults
-the path given in the client's [`cni_path`][] to find CNI plugin executables.
+1. [Install CNI Reference plugins](#install-cni-reference-plugins).
+1. [Configure bridge module to route traffic through iptables](#configure-bridge-module-to-route-traffic-through-iptables).
+1. [Verify cgroup controllers](#verify-cgroup-controllers).
+1. [Create a bridge mode configuration](#create-a-bridge-mode-configuration).
+1. [Configure Nomad clients](#configure-nomad-clients).
 
-## CNI configuration files
+### Install CNI reference plugins
 
-The CNI specification defines a network configuration format for administrators.
-It contains directives for both the orchestrator and the plugins to consume.
-At plugin execution time, this configuration format is interpreted by Nomad
-and transformed into arguments for the plugins.
+@include 'install/install-cni-plugins.mdx'
 
-Nomad reads the following extensions from the [`cni_config_dir`][]—
-`/opt/cni/config` by default:
+### Configure bridge module to route traffic through iptables
 
-* `.conflist` files are loaded as [network
-  configurations][cni_spec_net_config] that contain a list of plugin
-  configurations.
+@include 'install/bridge-iptables.mdx'
 
-* `.conf` and `.json` files are loaded as individual [plugin
-  configurations][cni_spec_plugin_config] for a specific network.
+### Verify cgroup controllers
 
-## Using CNI networks with Nomad jobs
+@include 'install/cgroup-controllers.mdx'
 
-To specify that a job should use a CNI network, set the task group's
-network [`mode`][] attribute to the value `cni/<your_cni_config_name>`.
-Nomad will then schedule the workload on client nodes that have fingerprinted a
-CNI configuration with the given name. For example, to use the configuration
-named `mynet`, you should set the task group's network mode to `cni/mynet`.
-Nodes that have a network configuration defining a network named `mynet` in
-their [`cni_config_dir`][] will be eligible to run the workload.
+## Create bridge mode configuration
```

Comment:
Something we discovered recently (ref https://github.com/hashicorp/nomad/issues/23583) is that the newest versions of some distros (ex. Ubuntu 24.04) aren't enabling bridge networking by default in some configurations (maybe only desktop editions? not sure). I've intended on adding a note to this section telling users to load the module (`sudo modprobe bridge`) if it isn't loaded. 

While we're touching this doc, can we add that here and in the install docs?

Reply:
Added to the partial that displays the Configure bridge module.. section